### PR TITLE
update yum cache upon adding nodesource repo

### PR DESCRIPTION
**Pull request by glaszig**
see #107

### CODE REVIEW

**Comment by glaszig** [comment]
bump

**Comment by glaszig** [comment]
testa failed. can i do anything about it? reason's not very clear to me.

**Comment by glaszig** [comment]
bump. still an issue.

**Comment by glaszig** [comment]
ping @geerlingguy

File: tasks/setup-RedHat.yml [hunk 1]
```diff
@@ -20,12 +20,19 @@
...
 - name: Add Nodesource repositories for Node.js (CentOS 7+).
   yum:
     name: "https://rpm.nodesource.com/{{ nodejs_rhel_rpm_dir }}/el/{{ ansible_distribution_major_version }}/{{ ansible_architecture }}/nodesource-release-el{{ ansible_distribution_major_version }}-1.noarch.rpm"
     state: present
   when: ansible_distribution_major_version|int >= 7
+  register: node_repo
+
+- name: Update package cache if repo was added.
+  yum: update_cache=yes
+  when: node_repo.changed
```
Comment (geerlingguy) [inline tasks/setup-RedHat.yml:34]:
```suggestion
  when: node_repo is changed
```

**Pull request merged**